src/foam/java/refinements.js

```diff
@@ -156,14 +165,13 @@ foam.CLASS({
     function generateSetter_() {
       return this.javaSetter ? this.javaSetter : `
```

Comment:
The intended goal of this section was to generate different setters (and getters) depending on whether you actually have a pre/postSet (or factory/expression) so we don't pay the costs for it.

Although the compiler may be smart enough to just eliminate the calls to pre/postset if they are no-ops so maybe its not necessary.  Would be nice if we could verify this some way.